--- nagios-naglite-1.6/index.php~	2013-10-28 23:52:47.000000000 +0200
+++ nagios-naglite-1.6/index.php	2015-08-06 18:14:13.011808961 +0300
@@ -18,10 +18,10 @@
  **/
 
 // Set file path to your nagios status log
-$statusFile = '/var/cache/nagios3/status.dat';
+$statusFile = '/var/lib/nagios/status.dat';
 
 // Objects file
-$objectsFile = '/var/cache/icinga/objects.cache';
+$objectsFile = '/var/lib/nagios/objects.cache';
 
 // Default refresh time in seconds
 $refresh = 10;
@@ -35,7 +35,7 @@
 
 // Enable fortune output
 $enableFortune = false;
-$fortunePath = "/usr/games/fortune";
+$fortunePath = "/usr/bin/fortune";
 
 // Uncomment to show custom heading
 //$nagliteHeading = '<Your Custom Heading>';
--- nagios-naglite-1.6/config.php~	2015-08-06 18:09:21.000000000 +0300
+++ nagios-naglite-1.6/config.php	2015-08-06 18:09:45.158645407 +0300
@@ -8,7 +8,7 @@
  **/
 
 // Set file path to your nagios status log
-// $statusFile = '/var/cache/nagios3/status.dat';
+// $statusFile = '/var/lib/nagios/status.dat';
 
 // Refresh time in seconds
 // $refresh = 10; 
@@ -30,7 +30,7 @@
 
 // Enable fortune output
 // $enableFortune = false;
-// $fortunePath = "/usr/games/fortune";
+// $fortunePath = "/usr/bin/fortune";
 
 // Set to callable to write custom heading by function
 // $nagliteHeading = function() { echo("<h2>This is Naglite3</h2>\n"); };
